frontend/packages/erd-core/src/features/erd/components/ERDRenderer/ERDRenderer.tsx

```diff
@@ -92,6 +93,26 @@ export const ERDRenderer: FC<Props> = ({
     document.cookie = `${PANEL_LAYOUT_COOKIE_NAME}=${JSON.stringify(sizes)}; path=/; max-age=${COOKIE_MAX_AGE}`
   }, [])
 
+  const useDeviceType = () => {
+    const [isMobile, setIsMobile] = useState(false)
+
+    useEffect(() => {
+      const handleResize = () => {
+        setIsMobile(window.innerWidth < 768)
+      }
+
+      handleResize()
+
+      window.addEventListener('resize', handleResize)
+
+      return () => window.removeEventListener('resize', handleResize)
+    }, [])
+
+    return { isMobile, isDesktop: !isMobile }
+  }
+
+  const { isMobile } = useDeviceType()
```

Comment:
Can you use this to identify the device? This is a more rigorous check.

https://github.com/liam-hq/liam/blob/poc-mh4gf-2/frontend/packages/erd-core/src/hooks/useIsTouchDevice.ts#L10